pkg/apimachinery/utils/verbs.go

```diff
@@ -19,4 +19,8 @@ const (
 	VerbDelete = "delete"
 	// VerbDelete is mapped from HTTP DELETE for collections
 	VerbDeleteCollection = "deletecollection"
+	// VerbGetPermissions is used when fetching resource specific permissions
+	VerbGetPermissions = "get_permissions"
```

Comment:
```suggestion
	VerbViewPermissions = "view_permissions"
```
maybe?? (or read) given that this may not resolve to a real "GET" but rather a filter on what permissions you can see.  

I don't feel strongly, but "view" implies the same thing without being tied to the HTTP verb